server/src/main/scala/za/co/absa/atum/server/api/RetrieveAwsSecret.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2021 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.atum.server.api
+
+
+import software.amazon.awssdk.auth.credentials.ProfileCredentialsProvider
+import software.amazon.awssdk.regions.Region
+import software.amazon.awssdk.services.secretsmanager.SecretsManagerClient
+import software.amazon.awssdk.services.secretsmanager.model.{GetSecretValueRequest, SecretsManagerException}
+
+
+/**
+ * Class implement the functionality of retrieving secret keys from aws secret manger service
+ */
+class RetrieveAwsSecret (profileCredentials: String = "npintdebdtools-sso") {
```

Comment:
What is the reason for the default value here?

Reply:
My bad, I forgot to remove it. I had that with the first test cases that I had. I'm removing it now.

Reply:
In fact, I wanted to put "default" as a value for that. I think it will be a good idea for those who will be using default profiles.